plugins/vst_base/RemoteVstPlugin.cpp

```diff
@@ -52,12 +52,39 @@
 #include <sched.h>
 #endif
 
+#ifndef NATIVE_LINUX_VST
 #include <wine/exception.h>
+#endif
 
 #endif
 
+#ifndef NATIVE_LINUX_VST
```

Comment:
Throughout this file are `#ifndef NATIVE_LINUX_VST` which as far as I can tell are just `#ifdef _WIN32` areas. Is there a reason it was done with `ifndef`?

Not saying we need to change it, just that it's odd to look a guard saying `LINUX` and seeing Windows stuff, then seeing it's a `ifndef` instead.

Reply:
PhysSong suggested changing a few of them to `_WIN32` in an earlier review, which I have done, but that was for code where a Windows/non-Windows distinction made sense. In a lot of other cases, the distinction is Windows/Linux/future platform X/etc., where there's less clarity to be gained by singling out Windows. At any rate, I'm hoping to factor platform-specific code out of this file after this PR is merged, which should hopefully make the choice redundant, so I'm inclined to leave this unless you feel strongly otherwise.

Reply:
Don't feel strongly about it, mainly strong about easy readability. The `ifndef` makes you have to do an extra negation step in your head for the code in the block it's surrounding. If you'll be refactoring later then it's all good. Just trying to be better readable. 😁